imports/ui/dialogs/create-report-dialog.jsx

```diff
@@ -48,16 +63,21 @@ class CreateReportDialog extends Component {
           </button>
         )}
         <form onSubmit={this.handleSubmit}>
-          <InputRow label='Report title' value={reportTitle}
+          <InputRow
+            label='Report title'
+            isFloatingLabelFixed
+            value={reportTitle}
+            isMultiLine
+            placeholder={`${this.creationTime()} ${this.props.unitName}`}
             onChange={(evt, val) => this.setState({reportTitle: val})}
             inpRef={el => { this.titleInputEl = el }}
             disabled={inProgress}
           />
           <div className='tr mt2'>
             <RaisedButton
               backgroundColor='var(--bondi-blue)'
-              disabled={!reportTitle || reportTitle.length === 0 || inProgress}
               type='submit'
+              onClick={() => this.populateTitle()}
```

Comment:
It would be better to handle the title default fallback in `handleSubmit` and not treat `populateTitle` as a side-effect of pressing the submit button